### PR TITLE
Add CODEOWNERS for rules_proto

### PR DESCRIPTION
**Pull request by Yannic**
Initial list of owners derived from https://docs.google.com/document/d/1Wai_okIK_NnhNJZai5nkTUvZVHdV5pp1YwNLFtWwpC4/edit?disco=AAAADI3o84E

### CODE REVIEW

**Comment by Yannic** [comment]
@laurentlb @lberki Can you review?

**Comment by laurentlb** [comment]
I'd rather not be in this owners file.

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* @hlopko @laurentlb @lberki
```
Comment (lberki) [inline CODEOWNERS]: Make this `hlopko`, `lberki` and `dbabkin` then.

Comment (Yannic) [reply CODEOWNERS]: Done! ptal (CI failure is fixed by #9)

Comment (lberki) [reply CODEOWNERS]: Merci!

**Pull request merged**